### PR TITLE
Merge main change color nav bar

### PR DESCRIPTION
**Pull request by arunraja-hub**

### CODE REVIEW

File: src/body.js [hunk 1]
```diff
@@ -29,7 +29,9 @@ import ReactDOM from 'react-dom';
   render() {
     return (
         <Routes>
+            {/* <Route path="/" element={<Navigate to='/home' />} /> */}
```
Comment (npqst) [inline src/body.js:32]: remove comments

File: src/components/body/navigation.css [hunk 2]
```diff
@@ -43,16 +43,33 @@
...
+    font-weight: bold;
+}
+
 .navigation-link:hover {
     color: #9a9c9e;
     text-decoration: none;
 }
 
+/* .current {
+    color:red;
```
Comment (npqst) [inline src/components/body/navigation.css:69]: remove comments

File: src/components/body/navigation.js [hunk 3]
```diff
@@ -3,8 +3,11 @@ import { NavLink } from "react-router-dom";
 
 import "./navigation.css";
 
-function Navigation() {
-    return (
+// const { pathname } = useLocation();
```
Comment (npqst) [inline src/components/body/navigation.js:6]: remove comments

File: src/components/introduction/introtext.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./introduction.css";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+
+// import Lipinski from "./lipinski_display.js";
+// import Descriptors from "./descriptor_display.js";
```
Comment (npqst) [inline src/components/introduction/introtext.js:7]: remove comments

**Review by anissa218 — APPROVED** [review]
looks great

File: src/components/introduction/introduction.css [hunk 5]
```diff
@@ -70,6 +71,7 @@
     text-align: justify;
     margin-bottom: 10pt;
     white-space:pre-line;
+    /* margin-left: 10%; */
```
[the remark this answers is not in the document]
Comment (arunraja-hub) [reply src/components/introduction/introduction.css:74]: removed

File: src/components/body/navigation.js [hunk 6]
```diff
@@ -13,19 +14,22 @@ function Navigation() {
...
                 <li className="navigation-item">
                   <NavLink className="navigation-link" to="/introduction">
                     Introduction
                   </NavLink>
+                  
                 </li>
+           
                 <li className="navigation-item">
-                  <NavLink className="navigation-link" to="/loadingpage">
+                  <NavLink className="navigation-link" to="/build">
```
[the remark this answers is not in the document]
Comment (arunraja-hub) [reply src/components/body/navigation.js:29]: yes skipping, or going back/forward doesnt seem to break the flow on my end

**Pull request merged**